pyresample/geometry.py

```diff
@@ -416,16 +422,6 @@ def get_edge_lonlats(self, vertices_per_side=None, frequency=None):
         blats = np.ma.concatenate(lats)
         return blons, blats
 
-    def get_edge_bbox_in_projection_coordinates(self, vertices_per_side: Optional[int] = None,
-                                                frequency: Optional[int] = None):
-        """Return the bounding box in projection coordinates."""
-        if frequency is not None:
-            warnings.warn("The `frequency` argument is pending deprecation, use `vertices_per_side` instead",
-                          PendingDeprecationWarning, stacklevel=2)
-        vertices_per_side = vertices_per_side or frequency
-        x, y = self._get_bbox_elements(self.get_proj_coords, vertices_per_side)
```

Comment:
@djhoese there is something going wrong with the CI. 
It does not run the tests on the code I pushed. 
It comply of a in-existing `_get_bbox_elements` method, but it's not called anymore at all in the repo !